pyproject.toml

```
@@ -55,6 +55,7 @@ sphinxcontrib-openapi = {version="^0.8.3", optional=true}
 mypy = {version="1.10.1", optional=true}
 requests = {version = "^2.32.3", optional = true}
 healpy = {version = "^1.18.0", optional = true}
+h5py = "^3.13.0"
```

```suggestion
h5py = {version = "^3.13.0", optional = true}
```
Should this be like the others in this optional section?

---

Definitely and added to the "test" section so it is installed in the testing runs.

---

will do! Thanks for that catch